resources/decks/schema.js

```diff
@@ -5,11 +5,17 @@ const deckSchema = joi.object({
     .string()
     .label('name')
     .required(),
+  tags: joi.array().items(joi.number().integer()),
+});
+
+const editDeckSchema = joi.object({
+  name: joi.string().label('name'),
   removeTags: joi.array().items(joi.number().integer()),
   addTags: joi.array().items(joi.number().integer()),
   tags: joi.array().items(joi.number().integer()),
```

Comment:
Do we still need the tags param on the editDeckSchema?

Reply:
Oh oops thank you hehe